cli/command_policy_show.go

```diff
@@ -160,14 +160,24 @@ func printFilesPolicy(p *policy.Policy, parents []*policy.Policy) {
 func printErrorHandlingPolicy(p *policy.Policy, parents []*policy.Policy) {
 	printStdout("Error handling policy:\n")
 
+	ignoreFile := false
+	if p.ErrorHandlingPolicy.IgnoreFileErrors != nil {
```

Comment:
nit: you can bind to local variable to avoid repeating long variable names.

ignoreFile := false
if ig := p.ErrorHandlingPolicy.IgnoreFileErrors; ig != nil {
  ignoreFile = *ig
}